src/order/order.providers.ts

```diff
@@ -0,0 +1,16 @@
+import { Connection } from 'typeorm';
+import { Order } from './order.entity';
+import { OrderItem } from './orderItem.entity';
+
+export const orderProviders = [
+  {
+    provide: 'ORDER_REPOSITORY',
+    useFactory: (connection: Connection) => connection.getRepository(Order),
+    inject: ['DATABASE_CONNECTION'],
+  },
+  {
+    provide: 'ORDERITEM_REPOSITORY',
```

Comment:
please raname it to ORDER_ITEM_REPOSITORY

Reply:
Could you create an enum or sth like this with keys used in DI container. That way you can avoid magic strings. 
See Notice in https://docs.nestjs.com/fundamentals/custom-providers#non-class-based-provider-tokens